api/db/migrate/20180202214000_migrate_user_roles_to_rolify.rb

```diff
@@ -0,0 +1,44 @@
+class MigrateUserRolesToRolify < ActiveRecord::Migration[5.0]
+  def up
+    say_with_time "Inserting global roles" do
+      execute <<~SQL.squish
+        INSERT INTO roles (name, created_at, updated_at)
+          VALUES
+          ('reader', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+          ('admin', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP)
+      SQL
+    end
+
+    say_with_time "Populating user_roles" do
+      execute <<~SQL.squish
+        INSERT INTO users_roles (user_id, role_id) (
+          SELECT u.id AS user_id, r.id AS role_id
+          FROM users u
+          INNER JOIN roles r ON r.name = u.role AND u.role IN ('admin', 'reader')
+            WHERE r.resource_id IS NULL AND r.resource_type IS NULL
+        )
+      SQL
+    end
+
+    remove_column :users, :role
```

Comment:
I see, it gets removed here. Should these three migrations be collapsed down into one so it's clearer what's happening?